jadx-gui/src/main/java/jadx/gui/plugins/context/CodePopupAction.java

```diff
@@ -34,12 +34,13 @@ private static class NodeAction extends JNodeAction {
 
 		public NodeAction(CodePopupAction data, CodeArea codeArea) {
 			super(data.name, codeArea);
+			setShortcutComponent(codeArea);
```

Comment:
This action class created from plugins and doesn't have associated ActionModel enum, so actionModel field set to null and cause NPE:
```java
java.lang.NullPointerException: Cannot invoke "jadx.gui.ui.action.ActionModel.getDefaultShortcut()" because "actionModel" is null
	at jadx.gui.settings.data.ShortcutsWrapper.get(ShortcutsWrapper.java:16)
	at jadx.gui.utils.shortcut.ShortcutsController.get(ShortcutsController.java:54)
	at jadx.gui.utils.shortcut.ShortcutsController.bindImmediate(ShortcutsController.java:79)
	at jadx.gui.ui.codearea.JNodePopupBuilder.add(JNodePopupBuilder.java:32)
	at jadx.gui.plugins.context.CommonGuiPluginsContext.appendPopupMenus(CommonGuiPluginsContext.java:71)
	at jadx.gui.ui.codearea.CodeArea.addMenuItems(CodeArea.java:133)
	at jadx.gui.ui.codearea.CodeArea.<init>(CodeArea.java:56)
	at jadx.gui.ui.codearea.ClassCodeContentPanel.<init>(ClassCodeContentPanel.java:48)
	at jadx.gui.treemodel.JClass.getContentPanel(JClass.java:119)
	at jadx.gui.ui.TabbedPane.getContentPanel(TabbedPane.java:404)
	at jadx.gui.ui.TabbedPane.showCode(TabbedPane.java:263)
	at jadx.gui.ui.TabbedPane.codeJump(TabbedPane.java:251)
	at jadx.gui.ui.TabbedPane.codeJump(TabbedPane.java:236)
	at jadx.gui.ui.MainWindow.nodeClickAction(MainWindow.java:801)
```
Can you mark `getActionModel()` as nullable and check all its use places?
Or maybe somehow opt out this class from using new shortcuts model at all.

Reply:
To reproduce this NPE, add [this script](https://github.com/skylot/jadx/blob/683cd76cc555f38808450674ff2f8653f1aa48de/jadx-plugins/jadx-script/examples/scripts/gui/custom_frida.jadx.kts#L15) to jadx project and try to open any class.

Reply:
Also fixed. Thanks :+1: